android/app/src/main/java/com/zulipmobile/notifications/NotificationIntentService.kt

```diff
@@ -1,17 +1,36 @@
 package com.zulipmobile.notifications
 
-import com.zulipmobile.notifications.onOpened
 import android.app.IntentService
 import android.content.Intent
 import com.zulipmobile.MainApplication
-import android.os.Bundle
+import android.util.Log
+import com.zulipmobile.MainActivity
 
 class NotificationIntentService : IntentService("NotificationIntentService") {
     override fun onHandleIntent(intent: Intent?) {
         val context = applicationContext as? MainApplication ?: return
         if (Intent.ACTION_VIEW == intent!!.action) {
             val data = intent.getBundleExtra(EXTRA_NOTIFICATION_DATA)
-            onOpened(context, data!!)
+            Log.d(TAG, "android notif: launching main activity")
```

Comment:
This commit and the next move this code around twice:

6fd1a19d2 android notif [nfc]: Inline NotificationUiManager.onOpened
 2 files changed, 23 insertions(+), 26 deletions(-)
2521e1df5 android notif: Remove NotificationIntentService
 2 files changed, 21 insertions(+), 50 deletions(-)

If squashed, the resulting diff is just slightly bigger than one of those:
 2 files changed, 21 insertions(+), 53 deletions(-)

That also keeps the move within one file, rather than moving to a different file and back. So I think it's clearest to squash those two together.

With that, the preceding commit:
3cf35647c android notif [nfc]: Rename variables to ease next refactor
 2 files changed, 6 insertions(+), 6 deletions(-)

also becomes unnecessary.